src/components/ColorPalette/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
```

Comment:
Do you mind moving styleguide specifics components to a different folder? Not in `components`, maybe `styleguide`?

Reply:
I don't think they should be going out of the `components` directory as they're `components` that are being rendered. What I can do is, create a subfolder `styleguide` under `components` and place the `ColorPalette`, `Typography` and `Icons` under here?

Also, on the side note. I still need to further break it down to be able to extract individual components with styles like `<Header />`. `<Caption />` etc. which I'll get to once I have better clarity on how it'll be used in other stories. Hence I've left it out in this PR. Feel free to add your thoughts on how well I can handle this scenario. Thanks! :)

Reply:
> create a subfolder styleguide

Sounds good!

> Hence I've left it out in this PR

Yes we can look at that later :)